packages/wallet/backend/src/user/service.ts

```diff
@@ -110,6 +117,53 @@ export class UserService implements IUserService {
     return !!user
   }
 
+  public async createDefaultAccount(email: string) {
+    const existingUser = await this.getByEmail(email)
+
+    if (existingUser) return
+
+    let defaultUser = this.env.DEFAULT_WALLET_ACCOUNT
+    const isBoutique = email === this.env.DEFAULT_BOUTIQUE_ACCOUNT.email
+    if (isBoutique) defaultUser = this.env.DEFAULT_BOUTIQUE_ACCOUNT
+
+    const args = {
+      ...defaultUser,
+      isEmailVerified: true
+    }
+
+    const createdUser = await User.query().insertAndFetch(args)
+
+    const defaultAccount = await this.accountService.createDefaultAccount(
+      createdUser.id,
+      isBoutique
+    )
+    if (defaultAccount) {
+      const typedArray = new Uint32Array(1)
+      getRandomValues(typedArray)
+      let walletAddressName = typedArray[0].toString(16)
+      if (isBoutique) walletAddressName = 'boutique'
+
+      const wallet = await this.walletAddressService.create({
+        accountId: defaultAccount.id,
+        walletAddressName,
+        publicName: 'Default Payment Pointer',
+        userId: createdUser.id,
+        isWM: false
+      })
+      await this.walletAddressService.registerKey(
+        createdUser.id,
+        defaultAccount.id,
+        wallet.id,
+        {
```

Comment:
keys should be registered only for the 'boutique' PP, right now we have the Developer accounts USD payment pointer having the same keys as the boutique, which is probably an error. So only save keys for 'boutique' PP